tests/unit/IOUUtilsTest.js

```diff
@@ -0,0 +1,139 @@
+import * as IOUUtils from '../../src/libs/IOUUtils';
+import * as ReportUtils from '../../src/libs/ReportUtils';
+
+let iouReport;
+let reportActions;
+const ownerEmail = 'owner@iou.com';
+const managerEmail = 'manager@iou.com';
+
+function createIOUReportAction(type, amount, currency, {IOUTransactionID, isOnline} = {}) {
+    let moneyRequestAction = ReportUtils.buildOptimisticIOUReportAction(
+        1,
+        type,
+        amount,
+        currency,
+        'Test comment',
+        [managerEmail],
+        '',
+        IOUTransactionID,
+        iouReport.reportID,
+    );
+
+    // Default is to create requests offline, if this is specified then we need to remove the pendingAction
+    if (isOnline) {
+        moneyRequestAction = {
```

Comment:
Can't we do `moneyRequestAction.pendingAction = null` or if we want to delete the key then `delete moneyReportAction.pendingAction`? Any specific reason we are copying it?

Reply:
Yeah totally, will update.

Reply:
Aaah, I see. It's fine then I guess.